migtests/tests/pg/case-sensitivity-reserved-words/add-pk-from-alter-to-create

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
```

Comment:
Instead of duplicating the file let's put this in the `scripts` dir and call it from there in scripts for all tests where required.

Reply:
Nit right now if it is easy to do then do it, else it's fine for now.

Reply:
Yes didn't want to get into this currently. Will do this separately.